### PR TITLE
Hide the single-table MySQL/PG cdc source and use the shared source instead.

### PR DESCRIPTION
**Pull request by fuyufjh**
## Info

- **Description**

Hide the single-table MySQL/PG cdc source and use the shared source instead.

Preparing for deprecating the single-table MySQL/PG cdc source.

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by ShanlanLi — APPROVED** [review]
LGTM, thanks

File: docs/guides/ingest-from-mysql-cdc.md [hunk 1]
```diff
@@ -164,16 +173,7 @@ WITH (
    connector='mysql-cdc',
```
Comment (StrikeW) [inline docs/guides/ingest-from-mysql-cdc.md]: `CREATE TABLE .. FROM source` only supports one property `snapshot='false'` to disable backfill.

File: docs/guides/ingest-from-postgres-cdc.md [hunk 2]
```diff
@@ -178,6 +164,22 @@ CREATE SOURCE [ IF NOT EXISTS ] source_name WITH (
...
 
+Syntax for creating a CDC table. Note that a primary key is required and must be consistent with the upstream table.
+
+```sql
+CREATE TABLE [ IF NOT EXISTS ] table_name (
+   column_name data_type PRIMARY KEY , ...
+   PRIMARY KEY ( column_name, ... )
+) 
+WITH (
+   connector='postgres-cdc',
```
Comment (StrikeW) [inline docs/guides/ingest-from-postgres-cdc.md]: ditto.  don't need to specify `connector='postgres-cdc'` for `CREATE TABLE`.

**Review by StrikeW — APPROVED** [review]
Rest lgtm

File: docs/guides/ingest-from-mysql-cdc.md [hunk 3]
```diff
@@ -164,16 +173,7 @@ WITH (
...
-
-```sql
-CREATE SOURCE [ IF NOT EXISTS ] source_name WITH (
-   connector='mysql-cdc',
-   <field>=<value>, ...
-);
+FROM source TABLE table_name;
 ```
 
 ### Connector parameters
```
Comment (emile-00) [inline docs/guides/ingest-from-mysql-cdc.md:178]: We'll need to specify if the parameters should be used with `CREATE SOURCE` or `CREATE TABLE`. Same for the PG CDC guide.

File: docs/guides/ingest-from-mysql-cdc.md (hunk 1 above)
Comment (fuyufjh) [reply docs/guides/ingest-from-mysql-cdc.md]: Thanks, my bad

**Review by StrikeW — APPROVED** [review]
LGTM

**Pull request merged**